packages/apollo/test/updaters/__snapshots__/addToList.test.ts.snap

```diff
@@ -4,13 +4,4 @@ exports[`addToList() errors if mutation data does not exist 1`] = `"Cannot find
 
 exports[`addToList() errors if property name is not an array 1`] = `"\\"getSomething\\" list \\"something.name\\" is not an array."`;
 
-exports[`addToList() errors if query does not exist in cache 1`] = `
-"Can't find field things on object {
-  \\"something\\": {
-    \\"type\\": \\"id\\",
-    \\"generated\\": false,
-    \\"id\\": \\"something:123\\",
-    \\"typename\\": \\"something\\"
-  }
-}."
-`;
+exports[`addToList() errors if query does not exist in cache 1`] = `"\\"otherQuery\\" list \\"something.things\\" is not an array."`;
```

Comment:
`cache.readQuery` does not throw an error if there's no results anymore, it returns `null` instead. That's why the other error is thrown here.